scripts/collect-stats.ts

```diff
@@ -178,5 +208,17 @@ class StatsCollector {
 const collector = new StatsCollector({
   org: 'withastro',
   token: process.env.GITHUB_TOKEN,
+  customCategories: {
+    "i18n": [
+      // Astro Docs content translations
+      "src/content/docs/!(en)/**/*",
+      // Astro Docs labels translations 
+      "src/i18n/!(en)/**/*",
+      // Starlight Docs content translations
+      "docs/src/content/docs/!(en)/**/*",
+      // Starlight package labels translations
+      "packages/starlight/translations/!(en.json)"
+     ],
+  }
```

Comment:
Should these be differentiated per repo something like this? It feels like you might not want to match the `i18n` paths of one repo in a different repo?
```suggestion
  customCategories: {
    i18n: {
      docs: [
        // Astro Docs content translations
        "src/content/docs/!(en)/**/*",
        // Astro Docs labels translations 
        "src/i18n/!(en)/**/*",
      ],
      starlight: [
        // Starlight Docs content translations
        "docs/src/content/docs/!(en)/**/*",
        // Starlight package labels translations
        "packages/starlight/translations/!(en.json)"
      ],
     },
  }
```
I was trying to think if there was a way to avoid the need for project-specific code here, i.e. get a blob of “raw” GitHub data in the collect step and then do the categorisation based on `achievements.config`, but I don’t think so. This approach still makes things easily configurable, so I’m happy with that!